### DISCUSSION TITLE
Is there a way go guarentee a layer to be always be rendered on top

### DISCUSSION THREAD

**Discussion by laijackylai** [visgl/deck.gl#6685] [Q&A] [answered]
I have been using Deck.gl for my personal project and I have a tiled terrain layer with a Geojson layer showing the coastline.
![Screenshot 2022-02-23 at 7 53 43 PM](https://user-images.githubusercontent.com/66239914/155314445-64e4615c-8fc3-4625-a6d8-c975076ce23b.png)
I want to always show the coastline despite the terrain layer with height, the temporary solution that I have found is this: https://deck.gl/docs/api-reference/core/layer#getpolygonoffset
by setting `getPolygonOffset` to `[0, -10000]` I was able to show most of the coast. However, some of the lines still end up clipping under the terrain layer, shown in the middle of the screenshot below
<img width="213" alt="Screenshot 2022-02-23 at 7 56 52 PM" src="https://user-images.githubusercontent.com/66239914/155314908-a272564b-6b44-4e7f-bfdc-23cd3301a02d.png">
The layers have already been ordered so that the Geojson layer renders after the Tile layer
`layers={[tiles, coastLine]}`
My question is that is there a way to guarantee that a layer will always render on top of the other layer, disregarding the z-value.

**Comment by ibgreen** [comment, marked as answer]
Try `depthTest: false` on the coastline: https://deck.gl/docs/developer-guide/tips-and-tricks#z-fighting-and-depth-testing

**Reply by laijackylai** [reply]
Thank you for your suggestion. It works now.
A better ["solution" #6644](https://github.com/visgl/deck.gl/issues/6644#issuecomment-1050717870) will be rendering the coastline along the surface of the generated terrain.
